lib/cloudflare-ddns.cpp

```diff
@@ -165,29 +188,29 @@ TACHI_NODISCARD int tachi_get_local_ip(
 	return 0;
 }
 
-TACHI_NODISCARD int tachi_get_record(
-	const char* TACHI_RESTRICT api_token,
-	const char* TACHI_RESTRICT zone_id,
-	const char* TACHI_RESTRICT record_name,
-	size_t record_ip_size, char* TACHI_RESTRICT record_ip,
-	size_t record_id_size, char* TACHI_RESTRICT record_id
-) TACHI_NOEXCEPT {
+DDNS_NODISCARD int ddns_get_record(
```

Comment:
Since you change the API anyway, maybe add a pointer to return whether the record is A or AAAA?

Reply:
Makes sense, thanks for pointing it out